scripts/get_excludes.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+
+
+import argparse
+import subprocess
+import sys
+import os
+import shlex
+
+
+def run_with_stdout(command):
+    assert isinstance(command, str)
+    print(f"Running: {command}", file=sys.stderr)
+    cmd_output = subprocess.check_output(command, shell=True, text=True)
+    return cmd_output
+
+
+def get_target_triple():
+    cmd_output = run_with_stdout("rustc -vV").strip().split("\n")
+    for s in cmd_output:
+        if s.startswith("host:"):
+            return s.split(" ")[1]
+    raise Exception("no target triple detected")
+
+
+def main():
+    target = get_target_triple()
+    print(f"identified target triple as: {target}", file=sys.stderr)
+    jq_query = (
```

Comment:
just for visibility ditto with: https://github.com/worldcoin/orb-software/pull/101/files#r1605355834